catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/SearchCommand.java

```diff
@@ -185,6 +193,19 @@ private Object executeSearchStore(Filter filter) throws Exception {
     return null;
   }
 
+  /** Returns the total hits matching {@param filter} or -1 if unknown */
+  private static long getHits(final Filter filter, final CatalogFacade catalogProvider) {
+    try {
+      final QueryImpl hitsQuery = new QueryImpl(filter);
+      hitsQuery.setRequestsTotalResultsCount(true);
+      hitsQuery.setPageSize(0);
```

Comment:
1 result So catalog framework doesn't automatically switch this to use the max page size